### ISSUE TITLE
Do not fail on non-standardized but already supported in browsers ecmascript features

### ISSUE THREAD

**Issue by Jokero** [webpack/webpack#4924]
**Do you want to request a *feature* or report a *bug*?**
It's not a bug, it's a feature

**What is the current behavior?**
I can't use webpack without babel for local development of application written using some ecmascript features which are not standardized yet but already supported by browsers.

**If the current behavior is a bug, please provide the steps to reproduce.**
For example, I already can use object spread operator `...` in chrome with flags and actually everything in my application works in chrome but webpack complains about `Unexpected token`.

`index.js`:
```js
const a = { a:1 };
const b = { b:2, ...a };
```

`webpack index.js bundle.js`:
```
ERROR in ./index.js
Module parse failed: /home/dmitry/test/index.js Unexpected token (2:17)
You may need an appropriate loader to handle this file type.
| const a = { a:1 };
| const b = { b:2, ...a };
```

**What is the expected behavior?**
I want to have a possibility to say webpack that all my js files are ok so the provided code should be bundled without errors.

**If this is a feature request, what is motivation or use case for changing the behavior?**
I don't want to slow down my local development by using unnecessary tools like babel

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
webpack 2.6.0
nodejs 6.9.2

**Comment by nyrosmith** [comment]
I think the Problem is that node 6.x does not support object spread syntax as it is stage3
https://stackoverflow.com/questions/40066731/node-v6-failing-on-object-spread

So I don't think it is in webpacks Hand, but I might be wrong...

**Comment by nyrosmith** [comment]
Sorry, it is not node, it's acorn that throws this error. acorn also only has stage 4 features implemented
From their readme:

> NOTE: Only 'stage 4' (finalized) ECMAScript features are being implemented by Acorn.

**Comment by Jokero** [comment]
If 3rd party module is used for parsing, it will be more complicated to implement this feature and connect with webpack. But anyway it's not critical at all, just nice to have feature

**Comment by TheLarkInn** [comment]
Correct, we made one exception for `import()` because of `System.import()` being deprecated to add a Stage 3 feature, however, we 99% of the time must wait for Stage 4 to implement new features (because of said acorn's same rules).

**Comment by TheLarkInn** [comment]
We could revisit this once it lands in stage four and would welcome someone to PR the update to acorn and parser changes if needed.
